src/pages/members/index.js

```diff
@@ -113,7 +115,32 @@ class Members extends React.PureComponent {
       };
     });
   };
-
+  /**
+   * Sort the member so that leader should be on top
+   */
+  sortMembers = (a, b) => {
+    const isALeader = this.checkForLeader(a);
+    const isBLeader = this.checkForLeader(b);
+    if (isALeader === isBLeader) {
+      return 0;
+    } else if (isALeader < isBLeader) {
+      return 1;
+    } else {
+      return -1;
+    }
+  };
+  /**
+   * Check the member for being a leader or not
+   */
+  checkForLeader = member => {
```

Comment:
this doesn't need to be a class function, could you move it out of the class?

Reply:
yes sure